### ISSUE TITLE
PR: "merge into" without organization/branch

### ISSUE THREAD

**Issue by melmus** [go-gitea/gitea#13582]
- Gitea version (or commit ref):
1.14.0+dev-106-g5b2096dd5
- Git version:
2.26.2
- Operating system:
Ubuntu 18.04.4 LTS
  
  
  
- Database (use `[x]`):
  - [X] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description

When I create PR, I want to see a branch "merge into", without organization/branch.
For example on screen: "applications:app_man_test_vp". I want to see: "app_man_test_vp"

## Screenshots
![gt1](https://user-images.githubusercontent.com/15018126/99230122-f83fb300-27ff-11eb-9fdc-933660649a9f.png)
![gt2](https://user-images.githubusercontent.com/15018126/99230128-f970e000-27ff-11eb-92f9-cf86bf3f3490.png)

**Comment by sgabenov** [comment]
Yep, this organization prefix seems to be redundant, as i am not able to merge into another repo/organization. And better to see the full name of branch, as it is trimmed by filter window width
